### PR TITLE
Add switch concept entry

### PR DESCRIPTION
**Pull request by braanj**
### Description

Add a concept entry: switch()

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/r/concepts/switch/switch.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  "August"    = "Summer",
+  "September" = "Autumn",
+  "October"   = "Autumn",
+  "November"  = "Autumn",
+  "December"  = "Winter",
+  "Unknown month"
+)
+```
+
+In this case, if the value of month does not match any of the specified cases, the default case "Unknown month" is returned and assigned to the variable season.
```
Comment (caupolicandiaz) [inline content/r/concepts/switch/switch.md:67]:
```suggestion
In this scenario, if the value of month does not match any of the specified cases, the default case `"Unknown month"` is returned and assigned to the variable `season`.
```

Comment (caupolicandiaz) [reply content/r/concepts/switch/switch.md:67]: looks like in this case it returns null in lieu of the default.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@braanj I've added my comments. Please update the entry, and tag me when you've finished making the changes. Thanks

File: content/r/concepts/switch/switch.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+## Syntax
+
+In the following code:
+
+- `expr` is the expression or value that you want to evaluate.
+- `case1`, `case2`, `case3`, and so on, are the possible cases or alternatives that you want to consider.
+
+The `expr` value is compared with each case value sequentially, and when a match is found, the corresponding expression or value associated with that case is returned.
+
+```pseudo
```
[the remark this answers is not in the document]
Comment (braanj) [reply content/r/concepts/switch/switch.md:20]: Done!

**Comment by braanj** [comment]
@caupolicandiaz  Changes made!

**Review by caupolicandiaz — APPROVED** [review]
@braanj Looks good. This is ready for a second review.

**Pull request merged**